packages/console/src/pages/Connectors/components/ConnectorName/index.tsx

```diff
@@ -15,13 +15,15 @@ import { ConnectorsTabs } from '@/consts/page-tabs';
 import ConnectorPlatformIcon from '@/icons/ConnectorPlatformIcon';
 import type { ConnectorGroup } from '@/types/connector';
 
+import DemoTag from './DemoTag';
 import * as styles from './index.module.scss';
 
 type Props = {
   connectorGroup: ConnectorGroup;
+  isDemo?: boolean;
 };
 
-const ConnectorName = ({ connectorGroup }: Props) => {
+const ConnectorName = ({ connectorGroup, isDemo = false }: Props) => {
```

Comment:
should `isDemo` be a connector property?

Reply:
Yes, this will be done in [LOG-5679](https://linear.app/silverhand/issue/LOG-5679)